opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -300,3 +303,130 @@ def test_collect(self):
         self.assertGreater(
             second_histogram.time_unix_nano, first_histogram.time_unix_nano
         )
+
+
+class TestConvertAggregationTemporality(TestCase):
+    """
+    Test aggregation temporality conversion algorithm
+    """
+
+    def test_mismatched_point_types(self):
+
+        current_point = Sum(0, 0, 0, AggregationTemporality.DELTA, False)
```

Comment:
It's much easier to read this if you pass named args

Reply:
Fixed